cli/src/main/java/bio/terra/pfb/JavaPfbCommand.java

```diff
@@ -10,6 +12,29 @@
     description = "A java implementation of pyPFB",
     versionProvider = PfbVersion.class)
 public class JavaPfbCommand implements Runnable {
+
+  @Parameters(index = "0", description = "command to run (Options include: show)")
+  private PfbCommand command;
+
+  @Parameters(
+      index = "1",
+      description =
+          "optional option to run for given command (Options include: schema, metadata, nodes)",
+      defaultValue = "tableRows")
```

Comment:
Is `tableRows` supposed to the default here?  I was expecting one of `schema`, `metadata`, or `nodes`?

Reply:
Thanks for calling this out -- I wasn't quite sure how to handle this case.  I'm trying to follow the same command syntax as used in pyPFB. In pyPFB, when there isn't a second argument for the "show" command, it defaults to display the table row data. So, I want this field to be optional and when it's not defined to be set to tableRows.

Reply:
Good point --added!